src/style.css

```diff
@@ -1,3 +1,78 @@
+* {
```

Comment:
Using the universal selector `*` to reset margins and paddings can negatively impact performance. Consider using more specific selectors or a CSS reset library.